### PR TITLE
docs: migrate GitHub wiki to docs

### PR DESCRIPTION
**Pull request by larsrickert**
Relates to #1638

Migrate our [GitHub wiki](https://github.com/SchwarzIT/onyx/wiki) to our VitePress docs and also update the content if its outdated.

Once this PR is merged and deployed, I will delete the wiki.

### CODE REVIEW

**Review by BoppLi — DISMISSED** [review]
thank you!

File: apps/docs/src/principles/technical-vision.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+   // ASSERT
+   ```
+
+## Things the future might hold
+
+The following points might be discussed when their use-case gets more concrete.
+
+- Offer package with test-utils
+- Measure the performance of components which contain a lot of content
+- We provide SSR support / compatibility
```
[the remark this answers is not in the document]
Comment (larsrickert) [reply apps/docs/src/principles/technical-vision.md:151]: There are still a few bugs open like #1521 so I'd keep it for now

File: apps/docs/src/principles/technical-vision.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+   - We use third-party dependencies sparingly. The need for each dependency must be well justified and documented in this wiki
+2. 📜 We use browser / native features as much as possible. We accept some drawbacks this decision might bring.
+3. 📜 Frequently used code is extracted into utils / composables
+4. 🔮 We provide custom eslint rules for principles and best practices where needed
+
+## Release process
+
+1. 📜 Every PR that affects the usage of one of our npm libraries must include a [changeset](https://github.com/changesets/changesets)
+2. 🔮 All packages use two release channels:
+   - `alpha`: automatically released on each merge to the `main` branch
```
[the remark this answers is not in the document]
Comment (larsrickert) [reply apps/docs/src/principles/technical-vision.md:25]: beta :)

File: apps/docs/src/principles/contributing.md [hunk 3]
```diff
@@ -14,8 +14,7 @@ It gives tips and guidelines on what should or must be considered when working w
 
 ## Prerequisites / Technical setup
 
-1. Install [Node.js](https://nodejs.org/en) version {{ nodeVersion }}.
-
+1. Install [Node.js](https://nodejs.org/en) version **{{ nodeVersion }}**.
```
Comment (BoppLi) [inline apps/docs/src/principles/contributing.md:17]: the double space is important so there will still be a line break:
```suggestion
1. Install [Node.js](https://nodejs.org/en) version **{{ nodeVersion }}**.  
```

**Review by BoppLi — DISMISSED** [review]

File: apps/docs/src/principles/contributing.md (hunk 3 above)
Comment (larsrickert) [reply apps/docs/src/principles/contributing.md]: I needed to use `<br />` because trailing spaces are removed by prettier

**Review by BoppLi — APPROVED** [review]

**Pull request merged**